src/ape/managers/chain.py

```diff
@@ -134,57 +134,6 @@ def query(
 
         return self.query_manager.query(query)
 
-    def range(
```

Comment:
It is imperative we delete this method?
Could we instead just to delegate to the query manager?

Reply:
not sure what you mean by delegate, I can leave range, and then pull the functionality to the DefaultQueryProvider, and then rename the function

Reply:
By delegate, I just mean call it from the query provider

Reply:
This was the original intent, yes. Perform a block query with all the fields selected, and then serialize it into a set of blocks and yield those.